### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by oddlittlebird**
Minor grammar fixes. Mostly correcting capitalization.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ $ amtool silence add alertname="Test_Alert" instance=~".+0"
 e48cb58a-0b17-49ba-b734-3585139b1d25
 ```
 
-View silences
+View silenced alerts:
```
Comment (simonpasquier) [inline README.md]: "silenced alerts" -> "silences", the command displays silences not alerts.

File: README.md [hunk 2]
```diff
@@ -289,7 +289,7 @@ $ amtool silence expire $(amtool silence query -q)
 
 Amtool allows a config file to specify some options for convenience. The default config file paths are `$HOME/.config/amtool/config.yml` or `/etc/amtool/config.yml`
 
-An example configfile might look like the following:
+An example config file might look like the following:
```
Comment (simonpasquier) [inline README.md]: config -> configuration

File: README.md [hunk 3]
```diff
@@ -313,7 +313,7 @@ receiver: team-X-pager
 Amtool allows you to visualize the routes of your configuration in form of text tree view.
 Also you can use it to test the routing by passing it label set of an alert
 and it prints out all receivers the alert would match ordered and separated by `,`.
-(If you use `--verify.receivers` amtool returns error code 1 on mismatch)
+(If you use `--verify.receivers` amtool returns error code 1 on mismatch.)
```
Comment (simonpasquier) [inline README.md]: no need for the additional dot

File: README.md [hunk 4]
```diff
@@ -376,11 +376,11 @@ alerting:
...
 > Important: Do not load balance traffic between Prometheus and its Alertmanagers, but instead point Prometheus to a list of all Alertmanagers. The Alertmanager implementation expects all alerts to be sent to all Alertmanagers to ensure high availability.
 
-### Disabling high availability
+### Turn off high availability
 
-If running Alertmanager in high availability mode is not desired, setting `--cluster.listen-address=` will prevent Alertmanager from listening to incoming peer requests.
+If running Alertmanager in high availability mode is not desired, setting `--cluster.listen-address=` prevents Alertmanager from listening to incoming peer requests.
 
-## Contributing to the Front-End
+## Contribute to the front end
```
Comment (simonpasquier) [inline README.md]: I would rename this section `## Contributing`, include the content from [CONTRIBUTING.md](https://github.com/prometheus/alertmanager/blob/master/CONTRIBUTING.md) before the existing text and remove `CONTRIBUTING.md`.

File: README.md [hunk 5]
```diff
@@ -376,11 +376,11 @@ alerting:
...
 
 Refer to [ui/app/CONTRIBUTING.md](ui/app/CONTRIBUTING.md).
```
Comment (simonpasquier) [inline README.md]: Could you also add a `## License` section like [here](https://github.com/prometheus/prometheus/blob/master/README.md#license)?

File: README.md [hunk 6]
```diff
@@ -33,7 +33,7 @@ $ GO15VENDOREXPERIMENT=1 go get github.com/prometheus/alertmanager/cmd/...
 $ alertmanager --config.file=<your_file>
 ```
 
-Or checkout the source code and build manually:
+Or check out the source code and build manually:
```
Comment (simonpasquier) [inline README.md]: "check out the source code" -> "clone the repository"

File: README.md [hunk 7]
```diff
@@ -289,7 +289,7 @@ $ amtool silence expire $(amtool silence query -q)
 
 Amtool allows a config file to specify some options for convenience. The default config file paths are `$HOME/.config/amtool/config.yml` or `/etc/amtool/config.yml`
```
Comment (simonpasquier) [inline README.md]: "config" -> "configuration"
Also could you check for "Amtool" occurrences and replace them with `amtool` where it makes sense?

File: README.md [hunk 8]
```diff
@@ -289,7 +289,7 @@ $ amtool silence expire $(amtool silence query -q)
```
Comment (simonpasquier) [inline README.md]: Rename section from "### Config" to "### Configuration"

**Comment by simonpasquier** [comment]
You'd need to sign your commits to comply with the [DCO check](https://github.com/prometheus/alertmanager/pull/2032/checks).

File: README.md (hunk 3 above)
Comment (oddlittlebird) [reply README.md]: You mean the period at the end of the sentence?

**Comment by oddlittlebird** [comment]
How do I sign my commits when I did not clone the repo? I just did quick changes in Git.

The other Grafana repos had a CLA signoff thing that just popped up and I clicked it.

File: README.md (hunk 3 above)
Comment (simonpasquier) [reply README.md]: yes

File: README.md [hunk 9]
```diff
@@ -382,13 +381,22 @@ If running Alertmanager in high availability mode is not desired, setting `--clu
...
-
 [hub]: https://hub.docker.com/r/prom/alertmanager/
 [circleci]: https://circleci.com/gh/prometheus/alertmanager
 [quay]: https://quay.io/repository/prometheus/alertmanager
+
+## License
+
+Apache License 2.0, see [LICENSE](https://github.com/prometheus/prometheus/blob/master/LICENSE).
+
+[travis]: https://travis-ci.org/prometheus/prometheus
```
Comment (simonpasquier) [inline README.md]: Remove the lines below please

**Comment by oddlittlebird** [comment]
All requested edits are complete. All I need to do is sign the DCO, which I will be happy to do once someone tells me how. As I said, I did not clone the repo, so not sure how to run a command line on a pull request that was never on my machine.

**Comment by stuartnelson3** [comment]
You have to clone the repo, then make the commits with `git commit -s`.

**Comment by oddlittlebird** [comment]
Looks like that did it.

**Comment by simonpasquier** [comment]
@oddlittlebird I've added a small [patch](https://github.com/prometheus/alertmanager/pull/2032/commits/e4e45e96fb7deb81a05285ee8767f6a7d987de2e) on top of your PR to update the Contributing section and remove `CONTRIBUTING.md`. Thanks for the work!

**Pull request merged**